### ISSUE TITLE
Some images not being fetched in gatsby-strapi query

### ISSUE THREAD

**Issue by lightlii** [newdesigncongress/wma-hk-web#149]
On all types (Post, event, collaborator etc) images are not coming through on certain items (though pattern is unclear). Seems to be fairly consistent across builds, eg [this item](https://kodak.ndc.tools/exhibitions/overriding-the-light-of-day/) never has a cover image despite having one [in the CMS](https://kodak-admin.ndc.tools/admin/content-manager/collectionType/api::event.event/126?plugins%5Bi18n%5D%5Blocale%5D=en).

**Comment by shibacomputer** [comment]
As part of the bug bounty programme, we made contact with a developer who was able to isolate the bug to two lines of code in a plugin for Gatsby's build pipeline. 

https://github.com/ptimson/gatsby-source-strapi-graphql/blob/master/build-query.js#L144

https://github.com/ptimson/gatsby-source-strapi-graphql/blob/master/build-query.js#L16

On both of these lines, the solution is to remove the hardcoded limit on pagination.

I'll update this when we have deployed a fix, which will likely be a fork of the library. The bounty will be paid out once this issue is closed. I'll keep the developer informed over the coming testing period.
